examples/ui-demo/src/app/config.tsx

```diff
@@ -79,9 +80,18 @@ export const DEFAULT_CONFIG: Config = {
 };
 
 export const queryClient = new QueryClient();
+const solanaConnection = new Connection(
+  `${
+    (global || window)?.location?.origin || "http://localhost:3000"
+  }/api/rpc/solana`,
```

Comment:
does the library not allow relative paths?

Reply:
Does not

Reply:
![image](https://github.com/user-attachments/assets/d90cde15-d21d-4738-bf09-2862947f2ca0)